viewer/viewer-awt/src/main/java/org/icepdf/ri/common/SwingViewBuilderFactory.java

```diff
@@ -0,0 +1,53 @@
+package org.icepdf.ri.common;
+
+import org.icepdf.ri.common.views.Controller;
+import org.icepdf.ri.common.views.DocumentViewController;
+import org.icepdf.ri.common.views.DocumentViewControllerImpl;
+import org.icepdf.ri.util.ViewerPropertiesManager;
+
+import java.awt.*;
+
+/**
+ * Factory for creating SwingViewBuilders
+ */
+public final class SwingViewBuilderFactory implements ViewBuilderFactory {
+
+    public static final SwingViewBuilderFactory INSTANCE = new SwingViewBuilderFactory();
```

Comment:
For API consistency can you convert this to a getInstance() method and make the constructor private.